color/src/rgba8.rs

```diff
@@ -24,8 +24,51 @@ pub struct Rgba8 {
     pub a: u8,
 }
 
+impl Rgba8 {
+    /// Returns the color as a packed value, with `r` as the most significant byte and
+    /// `a` the least.
+    #[must_use]
+    pub const fn to_u32(self) -> u32 {
+        ((self.r as u32) << 24) | ((self.g as u32) << 16) | ((self.b as u32) << 8) | self.a as u32
```

Comment:
Should this use [from_be_bytes](https://doc.rust-lang.org/std/primitive.u32.html#method.from_be_bytes) instead? That is also const.